x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Count.java

```diff
@@ -55,7 +55,7 @@ public class Count extends AggregateFunction implements ToAggregator, SurrogateE
             ),
             @Example(
                 description = "To count the number of times an expression returns `TRUE` use "
-                    + "a <<esql-where>> command to remove rows that shouldn't be included",
+                    + "a <<esql-where>> command to remove rows that shouldn’t be included",
```

Comment:
Is that `<<esql-where>>` correct?

Reply:
The MD syntax for this was very verbose and ugly, so for a list of common links, I added link replacement to the docs generating code instead. I did replace some links manually, but only a few.